llvm/docs/ReleaseNotes.md

```diff
@@ -443,6 +443,10 @@ Changes to LLDB
           _regexp-display   -- Evaluate an expression at every stop (see 'h...
   ```
 
+  * Minidumps generated by LLDB now support 64 bit memory, the capture of TLS variables, and capturing multiple signals or exceptions. Additionally to support 64 bit memory, Minidumps are now paged to disk while being written.
```

Comment:
Nested bullets would be a bit clearer.

* Minidumps generated by LLDB now support:
  * 64 bit memory (due to this, Minidumps are now paged to disk while being written).
  * ....etc.